mlflow/tensorflow/__init__.py

```diff
@@ -753,6 +763,29 @@ def autolog(
         warnings.warn("Could not log to MLflow. TensorFlow versions below 1.12 are not supported.")
         return
 
+    def _extract_tensor_as_numpy_input_example_slice(next_input):
+        """
+        Extracts first `INPUT_EXAMPLE_SAMPLE_ROWS` from the next_input, which can either be of
+        numpy array or tensor type.
+
+        :param next_input: an input of type `np.ndarray` or `tensorflow.Tensor`
+        :return: a slice (of limit `INPUT_EXAMPLE_SAMPLE_ROWS`)  of the input of type `np.ndarray`
+        """
+
+        input_feature_slice = None
+        if isinstance(next_input, tensorflow.Tensor):
+            input_feature_slice = np.take(next_input.numpy(), range(0, INPUT_EXAMPLE_SAMPLE_ROWS))
+        elif isinstance(next_input, np.ndarray):
+            input_feature_slice = np.take(next_input, range(0, INPUT_EXAMPLE_SAMPLE_ROWS))
+        else:
+            warnings.warn(
+                f"Tensorflow estimator doesn't support '{type(next_input)}' type for features"
+            )
```

Comment:
Can we update this message to make it clear that this type isn't support for *input example logging*? The rest of TensorFlow autologging capabilities will continue to work, so we don't want to confuse users.